src/components/elements/Table.js

```diff
@@ -40,24 +40,43 @@ const TableStyle = styled.table`
     padding-bottom: 10px;
   }
 
-  & tr:nth-child(even) td {
-    background-color: ${greyScale(9)};
-  }
 `;
 
-export const TableCell = ({ children, header }) => (
+export const TableCell = ({ children, header, }) => (
   <>{header ? <th>{children}</th> : <td>{children}</td>}</>
 );
 
-export const TableBody = styled.tbody``;
+export const TableSpanCell = styled.td`
+  grid-column: 1 / span 5;
+`
+
+export const TableRightAlignCell = styled.td`
+  text-align: right;
+  margin-right: 50px;
+`
+
+export const TableBody = ({children}) => (
+  <tbody>{children}</tbody>
```

Comment:
Not sure there's any benefit to this diff. Were you planning to extend the `TableBody` component? I'd rather keep the html `<table>` specific stuff as passthrough elements:
```
export const TableBody = styled.tbody``;
```